ci-operator/step-registry/telcov10n/verify-junit-reports-gz/telcov10n-verify-junit-reports-gz-commands.sh

```
@@ -0,0 +1,23 @@
+
+echo "Checking if the job should be skipped..."
+if [ -f "${SHARED_DIR}/skip.txt" ]; then
+  echo "Detected skip.txt file — skipping the job"
+  exit 0
+fi
+
+echo "Failed" >> ${SHARED_DIR}/job_status.txt
+
+export TESTS_DIR=$(mktemp -d)
+
+ls ${SHARED_DIR}/*.tar.gz | xargs -I {} tar xvf {} -C ${TESTS_DIR}
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2011)